.eslintrc.json

```diff
@@ -0,0 +1,165 @@
+    "browser": true,
+    "es2021": true,
+    "jest": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier",
+    "plugin:prettier/recommended"
+  ],
+  "overrides": [
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": [
+    "react",
+    "@typescript-eslint",
+    "react-hooks",
+    "prettier",
+    "feature-sliced-design",
+    "hooks",
+    "simple-import-sort"
+  ],
+  "rules": {
+    "react/react-in-jsx-scope": "off",
+    "react/display-name": "off",
+    "camelcase": "warn",
+    "spaced-comment": "error",
+    "quotes": ["error", "double"],
+    "no-duplicate-imports": "error",
+    "feature-sliced-design/relative-path-within-slice": [
+      "error", 
+      {
+        "alias": "@"
+      }
+    ],
+    "feature-sliced-design/public-api-slice-import": [
+      "error", 
+      { 
+        "alias": "@" 
+      }
+    ],
+    "feature-sliced-design/layers-hierarchy": [
+      "error", 
+      { 
+        "alias": "@",
+        "ignoredImports": [
+          "**/app/store"
+        ]
+      }
+    ],
+    "react/no-array-index-key": "error",
+    "no-console": "error",
+    "react/prop-types": "off",
+    "react/jsx-newline": "error",
+    "@typescript-eslint/no-unused-vars": "error",
+    "padding-line-between-statements": [
+      "error",
+      {
+        "blankLine": "always",
+        "prev": "*",
+        "next": [
+          "return",
+          "switch",
+          "case",
+          "default",
+          "if",
+          "for"
+        ]
+      }
+    ],
+
+    "hooks/sort": [
+      2,
+      {
+        "groups": [
+          "useLocation",
+          "useParams",
+          "useDispatch",
+          "useSelector",
+          "useState",
+          "useRef",
+          "useCallback",
+          "useLayoutEffect",
+          "useEffect"
+        ]
+      }
+    ],
+    "simple-import-sort/imports": [
+      "error",
+      {
+        "groups": [
+          // Side effect imports.
+          [
+            "^\\u0000"
+          ],
+          // Packages `react` related packages come first.
+          [
+            "^react"
+          ],
+          // Packages.
+          // Things that start with a letter (or digit or underscore).
+          [
+            "^\\w"
+          ],
+          // Packages.
+          // Things that start with a letter (or digit or underscore), or `@` followed by a letter.
+          [
+            "^@?\\w"
+          ],
+          // App
+          [
+            "^@app/shared"
+          ],
+          [
+            "^@app/entities"
+          ],
+          [
+            "^@app/features"
+          ],
+          [
+            "^@app/widgets"
+          ],
+          [
+            "^@app/pages"
+          ],
+          [
+            "^@app/processes"
+          ],
+          [
+            "^@app/app"
```

Comment:
It's not work. Because of `@app` in this case it's a `./src` folder. But in your config `./src` it's `@` aliases instead of `@app`. Please double check it